api/pkg/template/license.go

```diff
@@ -10,65 +10,89 @@ import (
 	"github.com/replicatedhq/embedded-cluster/pkg/release"
 )
 
+// Helper methods for direct access (used by tests and other code)
```

Comment:
these dont seem to be used in anything other than tests. are they really necessary?

Reply:
I'm reviewing, but you are probably right.